lib/cli/commands/openchannel.ts

```diff
@@ -3,22 +3,22 @@ import { callback, loadXudClient } from '../command';
 import { OpenChannelRequest } from '../../proto/xudrpc_pb';
 import { coinsToSats } from '../utils';
 
-export const command = 'openchannel <nodePubKey> <currency> <amount>';
+export const command = 'openchannel <node_key> <currency> <amount>';
 
-export const describe = 'open a payment channel';
+export const describe = 'open a payment channel with a peer';
 
 export const builder = {
-  nodePubKey: {
-    description: 'The node pub key of the peer to open the channel with',
+  node_key: {
```

Comment:
Think we also need to change:
`request.setNodePubKey(argv.nodePubKey);`
in this file.

Reply:
Fixed.